README.md

```diff
@@ -6,7 +6,7 @@ that expands its capabilities to gather Openshift Container Storage for informat
 
 ### Usage
 ```sh
-oc adm must-gather --image=quay.io/rhceph-dev/odf4-odf-must-gather-rhel9:latest-4.13
+oc adm must-gather --image=registry.redhat.io/odf4/odf-must-gather-rhel9:v4.13
```

Comment:
This is just a README, shouldn't we just use example instead of actual image path else we have to change it after every release.